packages/core/src/external/aws/lambda-logic/document-uploader.ts

```diff
@@ -49,7 +57,10 @@ export async function documentUploaderHandler(
   }
 
   // Get file info from the copied file
-  const { size, contentType } = await s3Utils.getFileInfoFromS3(destinationKey, destinationBucket);
+  const { size, contentType, eTag } = await s3Utils.getFileInfoFromS3(
```

Comment:
Would it be better to get file using `getObject`? And then getting the size and contentType from there. Using the eTag seems a bit hacky

Reply:
Do you mean that using the eTag as the hash is a bit hacky @Orta21? The etag is the MD5 hash of the file. 
Using getObject is also much less efficient than headObject (which is what getFileInfoFromS3 uses under the hood) because it downloads the object's body.